src/Sankey/SankeyLink/SankeyLink.tsx

```diff
@@ -170,6 +170,8 @@ export const SankeyLink: FC<Partial<SankeyLinkProps>> = ({
     }
   });
 
+  const ariaLabelData = useMemo(() => (`${(source as NodeExtra).title} → ${(target as NodeExtra).title}: ${formatValue(value)}`), [source, target, value]);
```

Comment:
Can we make a function for this and add tests? This isn't like the others here.

Reply:
Yes, it's unlike the others and very specific to the Sankey chart which has a different data structure. We can make a separate function for this but I don't think it's necessary given it's being used just here.